mage_ai/frontend/oracle/components/ProgressIcon/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Button from '@oracle/elements/Button';
+import Text from '@oracle/elements/Text';
+import { BORDER_RADIUS_SMALL } from '@oracle/styles/units/borders';
+import { MONO_FONT_FAMILY_REGULAR } from '@oracle/styles/fonts/primary';
+import { REGULAR } from '@oracle/styles/fonts/sizes';
+import { ArrowDown, ArrowUp } from '@oracle/icons';
+
+export type ProgressIconProps = {
+  children?: any;
+  percentage?: number;
+  danger?: boolean;
+  small?: boolean;
+};
+
+const ProgressIconStyle = styled.p<ProgressIconProps>`a
+  border-radius: ${BORDER_RADIUS_SMALL}px;
+  display: inline;
+  font-family: ${MONO_FONT_FAMILY_REGULAR};
+  font-size: ${REGULAR};
+  margin: 0;
+`;
+
+const ProgressIcon = ({
+  children,
+  ...props
+}: ProgressIconProps) => (
+  <ProgressIconStyle
+    {...props}
+  >
+    { props.danger
+      ?
+        <Button
+          afterIcon={<ArrowDown negative/>}
```

Comment:
you should be able to pass in a fill color for the icon

Reply:
I did that previously, if that's better I can revert to commit [1072b985d6c997c2ee01f5498d7b7cba1004e3b1](https://github.com/mage-ai/mage-ai/pull/347/commits/1072b985d6c997c2ee01f5498d7b7cba1004e3b1). I didn't know if I should be using the ThemeProps inside the component.